resources/public/api/conf/2.0/application.yaml

```diff
@@ -275,7 +275,15 @@ paths:
         Movements older than 31 days are archived, so any messages to related to them cannot be returned by the API.
 
         A successful response will include a Movement Reference Number (MRN) only after it has been allocated to a movement.
-
+        
+        Below is the set of all possible values returned in the message status field and their meaning:
+        <ul>
+          <li>Pending: the file has not yet been uploaded or we’re waiting for checks to complete</li>
+          <li>Processing: the file has been received by us and we’re processing the message</li>
+          <li>Success: the file has been processed successfully</li>
+          <li>Failed: there was an issue with the file and the message was not processed</li>
+        </ul>
+        

```

Comment:
This is useful but a few minor problems:

- The intro is longer than it needs to be.
- HTML tags are not needed - Markdown can suffice.
- GDS style guide advises against using contractions, such as "we're".
- status should have back ticks to highlight the fact that it is a parameter name.

Suggestion:
The `status` parameter can have the following values:

- Pending: the file has not yet been uploaded or checks are not yet completed
- Processing: the file has been received by us and the message is being processed
- Success: the file has been processed successfully
- Failed: there was a problem with the file and the message was not processed